src/targets/gpu/target.cpp

```diff
@@ -164,6 +164,15 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
     unsupported_fp8ocp_ops.insert("argmax");
     unsupported_fp8ocp_ops.insert("argmin");
 
+    // disable fp8 dot operations on gfx950
+    // hipblaslt does not support fp8 gemm with fp8 output yet
+    std::set<std::string> unsupported_gfx950_ops = {};
+    const auto device_name = trim(split_string(gpu::get_device_name(), ':').front());
+    if(starts_with(device_name, "gfx950"))
+    {
+        unsupported_gfx950_ops.insert("dot");
```

Comment:
why new set, why not just add to `unsupported_fp8ocp_ops`? This also reads a little funny, I'd expect a set called `unsupported_gfx950_ops` to include all unsupported ops on `gfx950` regardless of what the target device is.

Reply:
good idea, will change